piet/src/gradient.rs

```diff
@@ -123,12 +150,11 @@ pub struct RadialGradient {
 /// Mappings from the unit square into a non-square rectangle.
 #[derive(Debug, Clone)]
 pub enum ScaleMode {
-    /// The unit 1.0 is mapped to the smaller of width & height. All
-    /// values will be mapped without clipping, but the mapped item will
-    /// not cover the entire reect.
+    /// The unit 1.0 is mapped to the smaller of width & height, but the mapped
+    /// item may not cover the entire rectangle.
     Fit,
     /// The unit 1.0 is mapped to the larger of width & height; some
-    /// values on the other axis will be clipped.
+    /// values on the other axis may extend beyond the bounding rectangle.
```

Comment:
I think I'd say "unit" rather than "bounding" here. It doesn't actively bound (as in clipping or clamping the gradient values).

Reply:
that seems confusing, since the values as defined are within the unit; it's just that when we inflate them they may end up outside the target.